vmdb/db/migrate/20141126161823_convert_show_refresh_button_and_load_values_on_init_to_real_columns_for_dialog_fields.rb

```diff
@@ -0,0 +1,28 @@
+class ConvertShowRefreshButtonAndLoadValuesOnInitToRealColumnsForDialogFields < ActiveRecord::Migration
+  class DialogField < ActiveRecord::Base
+    serialize :options
```

Comment:
@eclarizio Not sure if I had a bad row in the db but I ran into an error here because options was coming back as nil.  You can avoid this by specifying the object type on the serialize line above.  This is how it is defined in the  model.
```ruby
serialize :options, Hash
```